pkg/schedule/operator/operator_controller.go

```diff
@@ -418,6 +418,16 @@ func (oc *Controller) PromoteWaitingOperator() {
 	}
 }
 
+// GetWopCount returns the count in the waiting list of a kind of scheduler.
+func (oc *Controller) GetWopCount(kind string) uint64 {
```

Comment:
How about using this function to check the concurrent operators ? `s.OpController.OperatorCount`